modules/api/static/schema-v0.0.4-dev.yml

```diff
@@ -79,13 +79,20 @@ components:
           properties:
             Stages:
               $ref: '#/components/schemas/InstrumentVisit'
-
     InstrumentVisit:
       type: object
       properties:
         Visit:
           $ref: '#/components/schemas/VisitStage'
-
+    Instrument:
```

Comment:
@xlecours i don't fully understand how this file works to update the schema. is just redefining the component schema like this enough?